mlflow/metrics/__init__.py

```diff
@@ -435,7 +424,6 @@ def f1_score() -> EvaluationMetric:
     "rougeL",
     "rougeLsum",
     "toxicity",
-    "make_genai_metric",
```

Comment:
Do we also need to remove "answer_similarity", "answer_correctness", etc. here?

Reply:
oof, great catch - fixed!